engine/test_modules.py

```diff
@@ -0,0 +1,12 @@
+# coding = utf-8
+# Create date: 2018-10-29
+# Author :Bowen Lee
+
+
+def test_modules(ros_kvm_for_kernel_parameters, kernel_parameters):
+    command = 'lsmod'
+    client = ros_kvm_for_kernel_parameters(kernel_parameters=kernel_parameters)
```

Comment:
The parameter "btrfs" should be passed in this function, and should not be put executor.py

Reply:
Please check the following changes